buildmodel/commands.go

```diff
@@ -221,6 +221,8 @@ func SubmitUpdatePR(f *PRFlags) error {
 		}
 		fmt.Printf("---- Skipping finding PR to update; no auth: %v\n", err)
 	} else {
+		*f.origin = auther.InsertAuth(*f.origin)
+		*f.to = auther.InsertAuth(*f.to)
```

Comment:
I looked at this again in the broader context, and this is actually a good opportunity to stop passing around the auth as unclearly. `auther.InsertAuth` should be used as late as possible (generally speaking) to make it clearer to future readers where sensitive information might be. This helps avoid inadvertently leaking it in a future change (e.g. in the PR desc).